model/src/main/proto/interaction_object.proto

```diff
@@ -95,6 +95,8 @@ message ImageWithRegions {
     }
 
     Region region = 2;
+    // Content description for selected image region.
```

Comment:
It might be nice to explain what this more precisely means. For example:

```// Content description to be read by screen readers when the associated region is clicked.``

Reply:
Updated